app/src/main/java/com/example/party_player/RetrofitClientInstance.kt

```diff
@@ -5,7 +5,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://api.spotify.com/v1/"
+    private const val BASE_URL = "https://api.spotify.com/v1/"
```

Comment:
This variable is never being changed, so it will seem to make more sense that it is a const variable.